examples/logger.py

```diff
@@ -63,10 +66,12 @@ def __init__(self, parent: Optional[QObject] = None):
         self.logEdit = QTextEdit(self)
         self.logEdit.setReadOnly(True)
         self.clearButton = QPushButton("Clear")
+        self.setLevelBox = QComboBox(self)
```

Comment:
`setLevelBox` seems like a method name which sets the levelBox.
Of course it is for _setting_ the level, but it also _shows_ the current level.
I think `levelBox` describes its role well enough.